cdap-ui/app/cdap/text/text-en.yaml

```diff
@@ -439,8 +442,9 @@ features:
         BrowserData:
           Content:
             directory: Directory
+            emptyBucket: No files or directories found in this bucket
             EmptymessageContainer:
-              suggestion1: your search
+              suggestion1: ' your search'
```

Comment:
why have a space before `your`?

Reply:
It's because we were concatenating it to another i18n property, whose value is `Clear`, and we needed to manually add this space so that it renders as `Clear your search` instead of `Clearyour search`. However it's not best practice to add spaces like this in i18n, so I have added a commit to switch to using CSS to add the space.